code/modules/projectiles/guns/energy/nuclear_vr.dm

```diff
@@ -0,0 +1,8 @@
+/*
+//	Virgo's very own file for overriding Polaris' eguns! Should include: Regular egun, Advanced egun, burst egun, mounted egun
+//
+//	NOTICE: All the egun sprites have to be mirrored in /icons/obj/eguns_vr.dmi because someone made them all child objects of the base thing
```

Comment:
Paths are good now, though not super-happy about this part. Trying to think of a way to circumvent this.

Reply:
I could just add an entry to each individual child object to point their icon back to the default one.